content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -206,8 +206,9 @@ point to the CA certificate and key.
 
 ### Managing the kubeadm drop-in file for the kubelet {#kubelet-drop-in}
 
-See the [setup of the kubelet drop-in file for systemd](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd)
-section for details about how the kubeadm package ships with configuration for how the kubelet should be run.
+The `kubeadm` package ships with a configuration file for running the `kubelet` by `systemd`. Note that the kubeadm CLI never touches this drop-in file. This drop-in file is part of the kubeadm DEB/RPM package.  This drop-in file is part of the `kubeadm` DEB/RPM package.
+
```

Comment:
@vorburger, looks like there are duplicate sentences -- end of line 209.
I added the link line as a list item, but feel free to not make this sentence a list.

Reply:
> @vorburger, looks like there are duplicate sentences -- end of line 209.

oups - sorry about that; removed it.

> I added the link line as a list item, but feel free to not make this sentence a list.

done